Release v1.9.6

philipjohn opened the pull request.

okupdatenumber commented:
My Dear Developer 👨‍💻👨‍💻👨‍💻

Please Fix error 

I have tried several times.  Transferred these files to the World Press but still the live blog do not work there. 

Because , The plugin becomes active but it is not being enabled in the WP Post. error is currently showing.  Please do it right.
![IMG_20200328_121202](https://user-images.githubusercontent.com/57313751/77816981-5ee75380-70ed-11ea-92d7-3aff4ff60512.jpg)

okupdatenumber commented:
A few years ago it worked correctly, but now due to an error, the entire plugin is not working

GaryJones commented:
@okupdatenumber Please open a new issue, with as much detail as possible - what WP version? What Liveblog version? Was it working correctly with a previous version? Have you disabled other plugins and custom code / used a fresh install to show it's not a conflict? Give us the minimum setup needed to be able to reproduce the issue ourselves, so that we can then try and debug and find a solution.

okupdatenumber commented:
https://github.com/Automattic/liveblog/issues/656#issue-589942960

philipjohn commented:
There have been further merges to master since this PR was opened, so I've rebased and updated the changelog to reflect the other bug fixes that can now be included in v1.9.6

GaryJones commented on CHANGELOG.md:
```diff
@@ -1,5 +1,15 @@
+* REST API routes require a permission_callback (#669)
+* Load CPT support later to avoid fatals with early use of WP_Query (#672)
+
+props [anigeluk], [david-binda], [GaryJones], [jeffersonrabb], [mslinnea], [philipjohn], [rebeccahum]
```
`anigeluk`, `jeffersonrabb`, and `mslinnea` all need corresponding links at the bottom of the file.

(There are also a couple of out-of-alphabetical-order ones at the very bottom - please move them to the correct order as well)

GaryJones commented on readme.txt:
```diff
@@ -3,8 +3,8 @@ Contributors: automattic, nbachiyski, batmoo, johnjamesjacoby, philipjohn
 Requires PHP: 5.6
-Tested up to: 4.9.8
-Stable tag: 1.9.5
+Tested up to: 5.3.2
```
Would be nice to have this tested on at least 5.7, if not a pre-5.8 release as well.

GaryJones replied:
@philipjohn Has it been tested on latest WP?

philipjohn replied:
It's now been tested with 5.8 so I've updated the tested tag in [cd8d219]

GaryJones commented on CHANGELOG.md:
```diff
@@ -1,5 +1,15 @@
 
+## 1.9.6
+
+* Revert #597 (#639)
```
Can we have a bit more detail on this one please? Someone reading this file not on GitHub, won't immediately have any sense of what was reverted.

GaryJones commented on CHANGELOG.md:
```diff
@@ -1,5 +1,15 @@
+
+* Revert #597 (#639)
+* Harden check when rendering media library (#652)
+* Clean comment cache after direct SQL queries. (#658)
```
This line has a full stop, but the others don't?

philipjohn replied:
I am nothing if not incosistent. [a6f6e6ccd2c492a2e74d7dbe8043f1f08a6f7743]

GaryJones commented:
Question on whether it has been tested on latest WP, and if so, could that `Tested up to` readme line be updated?

I also wonder whether we could do something like https://github.com/Parsely/wp-parsely/pull/187 first - GitHub action for auto-deploy to WordPress.org repo when adding a tag, readme.txt consolidated into README.md, screenshots numbering fixed and moved to `.wordpress-org` directory, changes extracted into CHANGELOG.md etc. All of these would ideally be done anyway, so doing it before this release just simplifies the release process (particularly the deploy to SVN).

Alternatively, if there's a time-pressure, then good to go on these 1.9.6 changes, and the other bits can be picked up before the next release.

philipjohn commented:
Auto-deploying would be great, agreed. I'll work on that separately tomorrow!

philipjohn commented:
FYI, auto-deployment in #675

GaryJones commented:
At least one of the failing tests hasn't been touched for 5-9 years, so I think this logic was always flawed. The [previous Travis configuration](https://github.com/Automattic/liveblog/pull/674/files?file-filters%5B%5D=.dist&file-filters%5B%5D=.json&file-filters%5B%5D=.sh&file-filters%5B%5D=.yml#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485) only tested up to PHP 7.0, so the issue was only revealed when the repo was switched to GitHub Actions, and up to PHP 7.4 was also tested.
